src/test/java/core/basesyntax/DataConverterTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitReaderImpl;
+import core.basesyntax.service.Reader;
+import core.basesyntax.service.ServiceShopForTestsException;
+import core.basesyntax.service.operation.DataConverter;
+import core.basesyntax.service.operation.DataConverterImpl;
+import core.basesyntax.service.operation.ServiceShopForTests;
+import core.basesyntax.service.operation.ServiceShopForTestsImpl;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+public class DataConverterTest {
+    public static final String FILE_FROM = "src/main/resources/FORTESTS.csv";
+    private ServiceShopForTests serviceShopForTests = new ServiceShopForTestsImpl();
```

Comment:
remove usage of ServiceShopForTests class here. You should test DataConverter here without using ServiceShopForTests